8.13.2 Release notes (#5058)
elastic/security-docs: docs/release-notes/8.13.asciidoc

[[release-notes-header-8.13.0]]
== 8.13

[discrete]
[[release-notes-8.13.2]]
=== 8.13.2

[discrete]
[[bug-fixes-8.13.2]]
==== Bug fixes

> * Fixes alert dashboard filters ({kibana-pull}179911[#179911]).

Author: @lgestc or @michaelolo24 can you elaborate a bit more on what was fixed? For example, was the issue that Alert table filters weren't retrieving the correct values when you used the inline filter in/filter out actions?

Reviewer: @nastasha-solomon https://github.com/elastic/kibana/issues/177906 this was the issue fixed, maybe it would be of help?

Author: Oh yeah, I saw that linked issue. There's quite a bit of information to sort through though, so I was hoping you could summarize the important parts. If the current summary is representative of the fix(es) that you made, we can move ahead with it. If there's more to note, we should expand the release note summary. 

And as a quick reminder, the release note summary should only be a sentence or two, for example:
Fixes a bug that broke the Alert table filters and stopped them from retrieving the correct values when you used the inline filter in/filter out actions.

Reviewer: That suggestion actually looks fine, I could go with it:)

Author: ok, I'll use the following summary then:

```suggestion
* Fixes a bug that broke the Alert table filters and stopped them from retrieving correct values if you used the inline filter in or out actions. ({kibana-pull}179911[#179911]).
```

> * Fixes a bug that broke the Alert table filters and stopped them from retrieving correct values if you used the inline filter in or out actions. ({kibana-pull}179911[#179911]).

Author: ```suggestion
* Fixes a bug that affected Alert table filters and stopped them from retrieving the correct values if you used the **Filter In** or **Filter Out** inline actions ({kibana-pull}179911[#179911]).
```

> * Fixes a bug that affected Alert table filters and stopped them from retrieving the correct values if you used the **Filter In** or **Filter Out** inline actions ({kibana-pull}179911[#179911]).

Reviewer 2: ```suggestion
* Fixes a bug that prevented Alert table filters from retrieving the correct values if you used the **Filter In** or **Filter Out** inline actions ({kibana-pull}179911[#179911]).
```
